lib/worker.js

```diff
@@ -15,6 +16,33 @@ const MINIMUM_ESLINT_VERSION = '7.0.0';
 const PATHS_CACHE = new Map();
 const ESLINT_CACHE = new Map();
 
+// TODO: There must be some cases where this logic is too naive, right?
+function descendsFrom (filePath, projectPath) {
+  return filePath.startsWith(projectPath);
```

Comment:
Adding a slash at the end of the project path ensures the last directory isn't the substring of one.

( e.g.
```js
filePath = "/path/to/the/file.txt"
projectPath = "/path/to/th"
// file is not in project
``` 
)

```suggestion
  return filePath.startsWith(projectPath.endsWith(Path.sep) ? projectPath : `${projectPath}${Path.sep}`);
```